tests/test_client.py

```diff
@@ -51,44 +66,52 @@
 slot = 1
 
 
+def _prepare_as_read_area(area: Area, size: int) -> Tuple[WordLen, Array[c_byte] | Array[c_int16] | Array[c_int32]]:
+    wordlen = area.wordlen()
+    usrdata = (wordlen.ctype * size)()
+    return wordlen, usrdata
+
+
+# noinspection PyTypeChecker,PyCallingNonCallable
 @pytest.mark.client
 class TestClient(unittest.TestCase):
     process = None
 
     @classmethod
-    def setUpClass(cls):
+    def setUpClass(cls) -> None:
```

Comment:
It's good that you noticed the lack of type annotations. I always forget about that :)
Maybe we should make the mypy settings stricter. Making the return value annotation mandatory